frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -980,7 +985,7 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
     const leftShiftedFields = _.reduce(
       fieldsToLeftShift,
       (fieldAccumulator, field) => {
-        const path = modifyArrayFieldPathIndex(field.path, (index) => index - 1);
+        const path = modifyArrayFieldPathIndex(field.path, (index) => (index || 1) - 1);
```

Comment:
fix type definition of `modifyArrayFieldPathIndex`. `index` should always be defined.